app/services/random_quotes.js

```diff
@@ -0,0 +1,19 @@
+const httpClient = require('../helpers/http_client');
+const logger = require('../logger');
+const config = require('../../config');
+
+const API_URL = config.common.external.quoteApiUrl;
+
+const getQuote = async (number = 'random', type = '') => {
+  try {
+    const typePrefix = type ? '/' : '';
+    const url = `${API_URL}/${number}${typePrefix}${type}`;
+
+    return await httpClient('get', url);
+  } catch (error) {
+    logger.error(`getQuote Error => ${error.message}`);
+    return new Error(`getQuote Error => ${error.message}`);
+  }
+};
+
+module.exports = { getQuote };
```

Comment:
I prefer use `exports. getQuote ` instance of `module.exports = { getQuote }`. In other functions you are use `exports`